tests/test_arcaflow_plugin_pcp.py

```diff
@@ -148,9 +148,12 @@ def test_functional(self):
                 print(f"==>> output_data is {output_data}")
 
                 self.assertEqual("success", output_id)
-                plugin.test_object_serialization(
-                    pcp_plugin.PerfOutput(output_data.pcp_output)
-                )
+                try:
+                    plugin.test_object_serialization(
+                        pcp_plugin.PerfOutput(output_data.pcp_output)
+                    )
+                except SystemExit:
+                    self.fail("Output failed schema validation")
```

Comment:
Catching `SystemExit` should generally be avoided.  Python (itself, and packages) typically provide other mechanisms for accomplishing anything that you would do by catching it explicitly.

For instance, [`test_object_serialization()` accepts three arguments](https://github.com/arcalot/arcaflow-plugin-sdk-python/blob/b6be00d9a10934c63720a2bfcefbb946db8f43b0/src/arcaflow_plugin_sdk/schema.py#L7084) but you're only providing one.  There is a `fail` parameter which allows you to provide for the disposition in failure cases.  So, something like this would be better:
```python
                plugin.test_object_serialization(
                    pcp_plugin.PerfOutput(output_data.pcp_output),
                    fail=self.fail,
                )
```
That is, in the event of a failure, instead of calling `sys.exit()` call [`unittest.TestCase.fail()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail) function (which is really what we would rather have happen).